python/neutron-understack/neutron_understack/nautobot.py

```diff
@@ -3,13 +3,14 @@
 from urllib.parse import urljoin
 
 import requests
+from neutron_lib import exceptions as exc
 from oslo_log import log
 from requests.models import HTTPError
 
 LOG = log.getLogger(__name__)
 
 
-class NautobotError(Exception):
+class NautobotError(exc.NeutronException):
```

Comment:
```
    message = "Nautobot error"
```


Instead of pass.

We should then have:

```
class NautobotNotFoundError(NautobotError):
    message = "%(obj)s not found in Nautobot. ref=%(ref)s"
```

Reply:
implemented